opam/cobol_data.opam

```diff
@@ -47,6 +47,7 @@ install: [
 depends: [
   "ocaml" {>= "4.14.0"}
   "dune" {>= "2.8.0"}
+  "zarith_stubs_js" {>= "0.17"}
```

Comment:
Hum… we need to investigate what impact that has. Ideally we'd find a way to avoid this dependency in the `opam` file (or have a flag that says the dependency is only valid for `js` target — not sure this is doable)?

Reply:
None I think, it only contains a `.js` file, so it won't be used unless compiled in JS mode